test/build.info

```diff
@@ -493,10 +485,9 @@ IF[{- !$disabled{tests} -}]
                      tls13encryptiontest wpackettest ctype_internal_test \
                      rdrand_sanitytest property_test ideatest \
                      rsa_sp800_56b_test bn_internal_test ecdsatest \
-                     rc2test rc4test rc5test hmactest ffc_internal_test \
```

Comment:
It's an internal test, we should keep it around and make it able to use the deprecated DH functions "internally"

Reply:
Bad rebase merge conflict fix :(

Reply:
Putting these back, fixes the CI tests.